src/testdir/test_channel.vim

```diff
@@ -2102,6 +2102,7 @@ endfunc
 func Test_job_start_in_timer()
   CheckFeature timers
   CheckFunction reltimefloat
+  CheckNotAsan
```

Comment:
Is this causing an asan error and that's why removed?

Reply:
This was an oversight as I cleaned up the branch. Earlier in my work I had to turn on a very, very slow ASAN option that interfered with the timing accuracy. That should no longer be an issue now that we have the debug packages + frame pointers.